python/plugins/processing/algs/gdal/GdalAlgorithmDialog.py

```diff
@@ -159,7 +159,7 @@ def parametersHaveChanged(self):
 
             try:
                 commands = self.algorithm().getConsoleCommands(
-                    parameters, context, feedback, executing=False
+                    parameters, context, feedback, executing=True
```

Comment:
This is wrong -- we are NOT executing here, so the existing code is correct and should not be changed